homeassistant/components/launch_library/sensor.py

```diff
@@ -86,15 +106,25 @@ async def async_setup_entry(
             NextLaunchSensor(
                 coordinator,
                 entry.entry_id,
+                SENSOR_DESCRIPTIONS[0],
                 name,
-                SENSOR_DESCRIPTION,
+            ),
+            LaunchTimeSensor(
+                coordinator,
+                entry.entry_id,
+                SENSOR_DESCRIPTIONS[1],
+            ),
+            LaunchProbabilitySensor(
+                coordinator,
+                entry.entry_id,
+                SENSOR_DESCRIPTIONS[2],
             ),
```

Comment:
Is there a better way to access the each description in the tuple? Or would a dict with keys be better instead?

Reply:
Do a dict for now